src/platform/packages/shared/kbn-esql-ast/src/commands_registry/index.ts

```diff
@@ -52,6 +53,7 @@ const baseCommands = [
   rowCommand,
   sortCommand,
   statsCommand,
+  inlineStatsCommand,
```

Comment:
Let's mark it is hidden for now as it is on snapshots releases still and we dont want to suggest it. When ES tells us that this is ready for tech preview we will change the flag

<img width="836" height="433" alt="image" src="https://github.com/user-attachments/assets/a689f414-3842-4e33-a2a2-7f52df9c4cea" />

Reply:
I removed this command from here. 

note: Is adding the flag hidden: true in the command object not enough?